pkg/requestid/requestid_test.go

```diff
@@ -22,6 +24,11 @@ func (m *mockTransport) RoundTrip(r *http.Request) (*http.Response, error) {
 	return nil, nil
 }
 
+func TestRequestID(t *testing.T) {
```

Comment:
Why did you add this? 
It's related only to test that run with ginkgo there is not `Describe` here so why have this definition here?

Reply:
I could change the whole infra to use Ginkgo but it is redundant. I only added the wrapper for Ginkgo, otherwise running this test package with ginkgo flags would fail. (Because the package doesn't know what is ginkgo)
As you can see in here - http://10.46.10.2:8080/job/assisted-service/job/PR-456/6/

Reply:
It's hacky, please add a comment or change the tests to Ginkgo.